pkg/destroy/powervs/dhcp.go

```diff
@@ -30,16 +30,16 @@ func (o *ClusterUninstaller) listDHCPNetworks() (cloudResources, error) {
 	result := []cloudResource{}
 	for _, dhcpServer = range dhcpServers {
 		if dhcpServer.Network == nil {
-			o.Logger.Debugf("listDHCPNetworks: DHCP has empty Network: %s\n", *dhcpServer.ID)
+			o.Logger.Debugf("listDHCPNetworks: DHCP has empty Network: %s", *dhcpServer.ID)
 			continue
 		}
 		if dhcpServer.Network.Name == nil {
-			o.Logger.Debugf("listDHCPNetworks: DHCP has empty Network.Name: %s\n", *dhcpServer.ID)
+			o.Logger.Debugf("listDHCPNetworks: DHCP has empty Network.Name: %s", *dhcpServer.ID)
 			continue
 		}
 
 		if strings.Contains(*dhcpServer.Network.Name, o.InfraID) {
-			o.Logger.Debugf("listDHCPNetworks: FOUND: %s (%s)\n", *dhcpServer.Network.Name, *dhcpServer.ID)
+			o.Logger.Debugf("listDHCPNetworks: FOUND: %s (%s)", *dhcpServer.Network.Name, *dhcpServer.ID)
```

Comment:
This is not related to the changes in the PR. It should at least be in a separate commit or on its own PR.

Reply:
The same comment about structured logging is also valid here. Consider using `o.Logger.WithField`.

Reply:
Thanks for the pointer!  I will keep that in mind for a future PR!

Reply:
The above comment was for the structured logging.  As for the \n, I was hoping to sneak them in and save a lot of work in having to do another PR which takes a crazy amount of time to get people to accept and push it.

Reply:
Then at least sneak them in in their own commit explaining their purpose. If the commit for removing the VPC is ever reverted, this would be reverted too.